Procurement/ViewModel/ItemHoverViewModelFactory.cs

```diff
@@ -70,13 +70,11 @@ internal static ItemHoverViewModel Create(Item item)
             if (item is Gem)
                 return new GemItemHoverViewModel(item);
 
-            if (item is Currency || item is Sextant || item is Essence || item is Fossil)
-                return new CurrencyItemHoverViewModel(item);
-
             if (item is Prophecy)
                 return new ProphecyItemHoverViewModel(item);
 
-            return new ItemHoverViewModel(item);
+            //Currency, sextant, essence, fossil, resonator, scarabs and splinters
+            return new CurrencyItemHoverViewModel(item);
```

Comment:
Are we sure it's safe to default to currency hover? What about if new types get added in the future?

Reply:
Your call, but the if to get to CurrencyItemHoverViewModel was getting out of hand. 7 items and no doubt more would be on the way.

Reply:
I'd rather do that and keep the order of precedence in tact. The class is probabaly due for a bit of a cleanup, as much as I hate them maybe a switch would be clearer with the number of conditionals, or separate hover view models for sextant/essence/fossil.

Reply:
I've restored it to a long IF statement as per your request.